test/client/before-test.js

```diff
@@ -138,6 +138,31 @@
         ok(passed);
     };
 
+    window.createIframeForTest = function (src, elLocation) {
```

Comment:
`window.createTestIframe (src)`. Now, to simplify let's drop the second parameter.
No sure that we will use this function for nested iframes.
Also let's try this function for several tests